src/style.css

```
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  border: 0;
+}
```

```suggestion
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
  border: 0;
}
```
- anything except `box-sizing` is forbidden to use in `*`
- you can move `margin, padding, border` to `body`